app/views/handlers/submission.py

```diff
@@ -56,14 +56,20 @@ def submit_questionnaire(self) -> None:
         message = json_dumps(payload)
 
         encrypted_message = encrypt(
-            message, current_app.eq["key_store"], KEY_PURPOSE_SUBMISSION  # type: ignore
+            message,
+            # type ignore: current_app can return empty Local Proxy. Similar to other files, this is ignored.
+            current_app.eq["key_store"],  # type: ignore
+            KEY_PURPOSE_SUBMISSION,
         )
-
+        # type ignore: metadata must be present at time of submission
         additional_metadata = get_receipting_metadata(self._metadata)  # type: ignore
 
+        # type ignore: current_app can return empty Local Proxy. Similar to other files, this is ignored.
         submitted = current_app.eq["submitter"].send_message(  # type: ignore
             encrypted_message,
+            # type ignore: case_id exists at this point
```

Comment:
I think it would be more correct to say that the metadata exists, in a single type ignore comment, as those are attributes that will always exist on the metadata provided the metadata is not None

Reply:
Each individual type ignore can also be removed, if you add a line like:

```py
metadata: MetadatProxy = self._metadata  # type: ignore
```

Then using metadata.case_id tx_id etc won't raise a mypy error..